pom.xml

```diff
@@ -8,7 +8,7 @@
         <version>22</version>
     </parent>
 
-    <groupId>io.takari.maven.plugins</groupId>
+    <groupId>com.facebook.presto</groupId>
```

Comment:
why this change?
Do we have `takari-maven-plugin` in `com.facebook.presto` maven group?
And how is that releated to this PR at all?